.infra/common.ts

```diff
@@ -297,6 +297,18 @@ export const workers: Worker[] = [
     topic: 'api.v1.post-visible',
     subscription: 'api.post-added-user-notification',
   },
+  {
+    topic: 'api.v1.squad-post-moderation-submitted',
```

Comment:
You also did streams?

Reply:
```suggestion
    topic: 'api.v1.source-post-moderation-submitted',
```

should we name it `source` so its generic? (also for others)

Reply:
Will create a PR for these 3 👀